### ISSUE TITLE
A: Create Digital City Map (DCM) Layer Files and Distribution Script

### ISSUE THREAD

**Issue by croswell81** [NYCPlanning/edm-overview#325]
Sprint C item: Finalize the metadata for Open Data posting (1/27/20)

Update script necessary for pulling DCM Feature Classes from sde@GISTRD
- [x] copies data to sde@GISPROD
- [x] packages and distributes to appropriate locations (closes #111)
- [x] creates layers (.lyr) and xmls for M-drive

**Layers needed for M-drive (internal):**
Group Layers:
- [x] Digital City Map [includes Mapped Streets, Street Treatment, Unmapped Street, Rail, Borough Boundaries]
  - [x] need values from TRD to create above layer
  - [x] filter out “Former_ROW” value [need filter values from TRD]
- [x] Pierhead and Bulkhead Lines [filter Feature Type "Bulkhead Line" or "Pierhead Line" and symbolize on Jurisdiction - confirm if "Water Edge" is needed and source w/ TRD]
- [x] Arterials and Major Streets [symbolize the two differently]
- [x] Local Law Names [symbolize the three; Intersections & Corners; Streets; and Parks, Plazas & Public Space

Individual Layers:
- [x] Digital City Map - Complete [no filters]
- [x] Mapped Streets [filter Feature Type - confirm w/ TRD]
- [x] Unmapped Streets [filter Feature Type - confirm w/ TRD]
- [x] Pierhead Line [filter Feature Type for "Pierhead Line"]
- [x] Bulkhead Line [filter Feature Type for "Bulkhead Line"]
- [x] Former_ROW [filter Type for "Right of Way" - confirm w/ TRD]
- [x] Digital City Map - Street Centerline [no filters]
- [x] Digital City Map - Street Centerline (Names Only) [symbolize without centerline visible, label should include street name and street width]
- [x] Paper Streets [filter - confirm w/ TRD]
- [x] Stair Streets [filter - confirm w/ TRD]
- [x] Record Street [filter Feature Type for "Record_St"] 
- [x] Local Law Names (Intersections & Corners) [points]
- [x] Local Law Names (Streets) [lines]
- [x] Local Law Names (Parks, Plazas & Public Space) [areas]
- [x] City Map Alterations (NYMI)
- [x] Final Section Map GRID [location = trd@GISTRD.sde\gistrd.TRD.DCM_FS_index]
- [x] Alteration Maps (georeferenced) [location = trd@GISTRD.sde\gistrd.TRD.DCM_Alt_maps_Phase2]

Note: Not including the following lyr files because the data is not ready or comprehenisve
Public Streets [filter - confirm w/ TRD]
Private Streets [filter - confirm w/ TRD]

**Comment by aferrar** [comment]
*12/10/19*
Script complete. Waiting on review of Feature Classes, Shapefiles, and Layers from Andrew E (TRD) and Matt C (GIS).

**Comment by aferrar** [comment]
**12/13/2019**

Script and outputs are complete. Waiting for review from @croswell81 and Andrew English from TRD. Reached out to TRD on 12/10/2019.

**Comment by aferrar** [comment]
**2/11/2020**

Was serving as placeholder for some metadata cleaning that needs to occur. Closing this issue since the script and associated layer files are generated, A new issue will be created, associated with sprint D, for any additional work that needs to be completed.

**Comment by aferrar** [comment]
**2/11/2020**

Was serving as placeholder for some metadata cleaning that needs to occur. Closing this issue since the script and associated layer files are generated, A new issue will be created, associated with sprint D, for any additional work that needs to be completed.
